src/components/secrets/Secret.tsx

```diff
@@ -299,7 +308,7 @@ export const SecretForm: React.FC<SecretFormProps> = function (props) {
             return
         }
 
-        if (!/^[-.a-zA-Z0-9]+$/.test(configName.value)) {
+        if (!secretNameRegex.test(configName.value)) {
             setName({ value: configName.value, error: 'Name must be of format /^[-.a-zA-Z0-9]+$/' })
```

Comment:
expand this

Reply:
expanded for regex `/^[a-z][a-z0-9-.]+[a-z]$/ `
to `Name must start and end with an alphanumeric character. It can contain only lowercase alphanumeric characters, '-' or '.'`